test/test-load.c

```diff
@@ -273,6 +289,9 @@ int main(int argc, char **argv)
 	g_test_add("/load/056a:WACf004", struct fixture, NULL,
 		   fixture_setup, test_wacf004,
 		   fixture_teardown);
+	g_test_add("/load/056a:48000", struct fixture, NULL,
```

Comment:
I guess "/load/056a:48000" should be "/load/056a:4800". Except that, the logic sounds right to me.

Reply:
fixed, thanks